pkg/decision/composite_experiment_service.go

```diff
@@ -47,10 +53,14 @@ func (s CompositeExperimentService) GetDecision(decisionContext ExperimentDecisi
 
 	// Run through the various decision services until we get a decision
 	for _, experimentService := range s.experimentServices {
-		if decision, err := experimentService.GetDecision(decisionContext, userContext); decision.Variation != nil {
+		decision, err := experimentService.GetDecision(decisionContext, userContext)
+		if err != nil {
+			ceLogger.Debug(fmt.Sprintf("%v", err))
+		}
+		if decision.Variation != nil && err == nil {
 			return decision, err
 		}
 	}
 
-	return experimentDecision, nil
+	return experimentDecision, errors.New("no decision was made")
```

Comment:
Should we log this error or should the parent `composite_service` log it?

Reply:
If I'm reading correctly, the client will end up logging this error, and the same is true for the composite feature service.

Reply:
Ah got it, ship away then!